fe/fe-core/src/test/java/com/starrocks/server/SharedDataStorageVolumeMgrTest.java

```diff
@@ -878,7 +878,7 @@ public void testGetStorageVolumeNameOfDb() throws DdlException, AlreadyExistsExc
     }
 
     @Test
-    public void testCreateHDFS() throws DdlException, AlreadyExistsException {
+    public void testCreateHDFS() throws DdlException, AlreadyExistsException, MetaNotFoundException {
```

Comment:
This should be legacy test case design issue, these exceptions shall not be added to a test method declaration.

All the statement that may throw these exception shall be wrapped with `Assertions.assertNotThrow(() -> {...});`